include/paths.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * paths.h compatibility shim
+ */
+
+#ifdef HAVE_PATHS_H
+
+#include_next <paths.h>
+
+#else
+
+#ifndef LIBCOMPAT_PATHS_H
+#define LIBCOMPAT_PATHS_H
+
+#define    _PATH_DEVNULL   "/dev/null"
```

Comment:
I suspect we could replace this simply with "/dev/null" upstream and remove the need for this.